statsmodels/base/_parameter_inference.py

```diff
@@ -220,7 +214,11 @@ def score_test(self, exog_extra=None, params_constrained=None,
         score = score_obs.sum(0)
         hessian_factor = model.hessian_factor(params_constrained,
                                               **hess_kwd)
-        hessian = -np.dot(ex.T * hessian_factor, ex)
+        # see #4714
+        from statsmodels.genmod.generalized_linear_model import GLM
+        if isinstance(model, GLM):
```

Comment:
A better approach would be the have models defind an attribute, e.g. `hessain_factor` that would be 1 by default and could be overridden when necessary, e.g., by GLM, rather than encodign model-class-specific information in a base function.

Reply:
That would be better if this were a permanent feature.
However, I hope that we can make this consistent across models and then we just need to delete these lines here.

I have not checked yet how this works in other models like OLS/WLS and RLM, or models in survival.